package.json

```diff
@@ -1,43 +1,49 @@
 {
-  "name": "satcheljs-suite",
```

Comment:
Any idea why every line is showing up as changed?  Something to do with line endings, maybe?  Can you figure out what it is so we'll have a clean diff?